src/web/lib/components/ThemeColorsEditor/index.scss

```diff
@@ -96,3 +103,25 @@
   box-shadow: none !important;
   border-radius: 0 !important;
 }
+
+.use-default {
+  display: inline-flex;
+  background-color: white;
+  border: none;
+  cursor: pointer;
+  margin-left: 8px;
+}
+
+.use-default span {
```

Comment:
The `span` appears to only exist for styling purposes. If that's the case, consider using the `::before` pseudo-element instead.